Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/EventType/EventTypeForm.tsx

```diff
@@ -90,6 +90,7 @@ export default function EventTypeForm(props: IProps) {
                     <Input<OpenXDA.Types.EventType> Record={props.Record} Field={'Name'} Label={'Name'} Disabled={true} Help={'Name cannot be changed. To change what is displayed in various visualization apps, use Label.'} Valid={() => true} Setter={() => { } } />
                     <Input<OpenXDA.Types.EventType> Record={props.Record} Field={'Category'} Label={'Category'} Feedback={'Category must be less than 50 characters.'} Valid={Valid} Setter={props.Setter} />
                     <CheckBox<OpenXDA.Types.EventType> Label={'Show in User Interfaces'} Record={props.Record} Field={'ShowInFilter'} Setter={props.Setter} />
+                    <br/>
```

Comment:
not saying this doesn't work but it feels like this should be a gemstone fix... otherwise this could be a problem in other places. We probably need to adjust margins in `Checkbox`

Reply:
Addressed as per our conversation